_includes/selector.html

```diff
@@ -392,9 +392,9 @@
             packages: ["Standard", "Choose Specific Packages"],
             additional_pip_packages: ["cuDF", "dask-cuDF", "cuML", "cuGraph", "cuSpatial/cuProj", "cuxfilter", "cuCIM", "RAFT", "cuVS"],
             additional_rapids_packages: ["cuDF", "cuML", "cuGraph", "cuSpatial/cuProj", "cuxfilter", "cuCIM", "RAFT", "cuVS"],
-            additional_packages: ["JupyterLab", "Plotly Dash", "Graphistry", "TensorFlow", "Xarray-Spatial", "PyTorch"],
+            additional_packages: ["Graphistry", "JupyterLab", "NetworkX + nx-cugraph", "Plotly Dash", "PyTorch", "TensorFlow", "Xarray-Spatial"],
             note_prefix: "<i class='fas fa-info-circle text-blue'></i>",
-            rapids_meta_pkgs: ["cuDF", "cuML", "cuGraph", "cuSpatial", "cuProj", "cuxfilter", "cuCIM", "RAFT"],
+            rapids_meta_pkgs: ["cuDF", "cuML", "cuGraph", "cuSpatial", "cuProj", "cuxfilter", "cuCIM", "nx-cugraph", "RAFT"],
```

Comment:
Let's put `nx-cugraph` next to `cuGraph`. It is more related to cuGraph than cuCIM or RAFT. Also we happen to be missing cuVS from this list, so I fixed that (I used the same order as the buttons except for nx-cugraph because it doesn't have a package button, but I think this should be fine).
```suggestion
            rapids_meta_pkgs: ["cuDF", "cuML", "cuGraph", "nx-cugraph", "cuSpatial", "cuProj", "cuxfilter", "cuCIM", "RAFT", "cuVS"],
```